config/jasmine.json

```diff
@@ -8,6 +8,7 @@
   ],
   "helpers": [
     "node/**/*-helper*.js",
-    "any/**/*-helper*.js"
+    "any/**/*-helper*.js",
+    "../node_modules/foam2/test/helpers/testcontext.js"
```

Comment:
I recommend using trailing commas, if possible. It makes the diff for appending a new entry to the list easier to review.